packages/@sanity/preview/src/components/RenderPreviewSnapshot.tsx

```diff
@@ -28,13 +28,14 @@ export default function RenderPreviewSnapshot(props: Props) {
 
   // TODO: Bjoerge: Check for image type with "is()"
   const renderAsBlockImage = layout === 'block' && type && type.name === 'image'
-
+  const typeName = snapshot?._type
+  const icon = (type.to && (type.to || []).find(t => t.name === typeName)?.icon) || (type && type.icon)
```

Comment:
Can `type.to` be something else than an array?

Reply:
It can also be an object I believe?

Reply:
When it's just a reference to a single type: `to: {type: 'myType'}`

Reply:
That works? Well, TIL!

Reply:
There's a lot of it in the test-studio schemas 😅 It was also news to me

Reply:
Although the schema accepts an object as value for `to`, I'm 99% sure it's normalized to an array after it's been processed/parsed.